### DISCUSSION TITLE
付费后专业版变成免费版？？？

### DISCUSSION THREAD

**Discussion by Janslo1990** [getlantern/lantern-forum-cn#510] [General]
我从3月13日付款后到现在，专业版一直变成免费版，一直头疼无法使用，之前是老用户用了有些年觉得还行，但是今年的这个操作我觉得非常失望，希望有人解决这个问题

**Comment by MaxMKMax** [comment]
@Janslo1990 可能是您没收到激活码来登入专业版. 请您提供我账号邮箱和支付单据. 我帮您提交上去然后来解决您的问题.

也顺便帮我提交各应用内报告然后提供我 8 位数票号. 您提交完毕后票号会出现在邮箱. 谢谢!

**Reply by Janslo1990** [reply]
632979426@qq.com
![微信图片_20240516011336](https://github.com/getlantern/lantern-forum-cn/assets/169930355/0e1167cf-62cf-4ea8-ba9d-5a41b53519db)

**Comment by Janslo1990** [comment]
这样可以了吗

**Reply by MaxMKMax** [reply]
@Janslo1990 好了. 帮您提交上去了. 不好意思影响了您的体验感. 谢谢您支持蓝灯. 对您这位老用户感激万分!

**Reply by MaxMKMax** [reply]
@Janslo1990 您好. 这是您专属的激活码:
YRG8B-BGYB7-BM3V8-69RK4-2MC3W
